requirements_test.txt

```diff
@@ -0,0 +1,9 @@
+-r requirements.txt
+black
+codespell
+flake8
+isort
+mypy
+pytest
+pyupgrade
+psycopg2
```

Comment:
@anandology Should `psycopg2` be moved to `requirements.txt` or is Postgres optional at runtime?